packages/ui/.babelrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "sourceType": "unambiguous",
+    "presets": [
+        ["@babel/preset-env"],
```

Comment:
We can avoid the `[]` since we are not passing any configuration object to the `preset-env`.

Reply:
Addressed in f087b5c11cffb39da92e403c638b77059e5d26bc